### ISSUE TITLE
Score not found error/info.url error (command-line tool can no longer download files)

### ISSUE THREAD

**Issue by QuynhVir** [LibreScore/dl-librescore#44]
I'm getting "Score not found" error on every score, I was still able to use it last week

**Comment by PeterNjeim** [comment]
Lots of debugging later, this is due to [Cloudflare's anti-scraping cookie](https://developers.cloudflare.com/fundamentals/reference/policies-compliances/cloudflare-cookies/#__cf_bm-cookie-for-cloudflare-bot-products): `__cf_bm`. It'll take a while for me to fix this, so in the meantime please use the userscript

**Comment by allandiego** [comment]
> Lots of debugging later, this is due to [Cloudflare's anti-scraping cookie](https://developers.cloudflare.com/fundamentals/reference/policies-compliances/cloudflare-cookies/#__cf_bm-cookie-for-cloudflare-bot-products): `__cf_bm`. It'll take a while for me to fix this, so in the meantime please use the [Userscript](https://github.com/LibreScore/dl-librescore?tab=readme-ov-file#userscript)

In addition to changing the parameters in the url and the cloudflare anti-bot that blocks direct requests without being from within a browser, it appears that musescore modified the hardcoded authorization tokens, its now generating a random 4 characters token for the api/jmuse requests

**Comment by PeterNjeim** [comment]
That's a known issue and I was working on it both when resolving #45 and yesterday when I noticed that PDF's stopped downloading. I tried to fix it but due to the original hacky implementation (there's a comment where they admit it's hacky lol), I couldn't easily do it. I might need to rewrite this section of code from scratch to fix it. Definitely today or tomorrow this new issue should be resolved
